pkg/webhook/config.go

```diff
@@ -88,6 +89,10 @@ func (cfg *Config) Validate() error {
 		return err
 	}
 
+	if cfg.RunnerLabel == "" {
```

Comment:
```suggestion
	if strings.TrimSpace(cfg.RunnerLabel )== "" {
```
I'm a fan of Java's [String.isBlank()](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#isBlank()) though not a requirement to do it this way